src/main/java/frc/robot/controllers/OperatorJoystick.java

```diff
@@ -25,11 +33,42 @@ public boolean getButtonPressed(int buttonID) {
         return joystick.getRawButtonPressed(buttonID);
     }
 
+    public boolean getButtonReleased(int buttonID) {
+        return joystick.getRawButtonReleased(buttonID);
+    }
 
     public boolean getAxisPressed(int axisID) {
         return joystick.getRawAxis(axisID) > 0.5;
     }
 
+    public boolean getAxisReleased(int axisID) {
```

Comment:
I'm like 80% sure this doesn't do what you want it to do. If the axis is negative, as in the other direction, it will return true.